Dockerfile

```diff
@@ -23,7 +23,4 @@ RUN chown -R app:app ./
 
 USER app
 
-VOLUME /code
-WORKDIR /code
```

Comment:
IIRC this is a requirement of the SPEC. I'm not sure we're in love with that requirement, or if _all_ our engines follow it, but while it's there we might as well not make changes that invalidate engines that _are_ currently in compliance.